packml_ros/launch/packml_ros.launch

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<launch>
+
+    <arg name="stats_publish_period" default="1.0" doc="Delay of stats messages publishing in seconds. 0 or less will not publish at all"/>
```

Comment:
Delay might be misleading in this context

Reply:
Change name and doc wording to rate .